### PR TITLE
Landing category questions

### PR DESCRIPTION
**Pull request by AagmanSuri**

### CODE REVIEW

File: Pages/BigBangQuiz/BigBangAnswers.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+
+        </div>
+
+        <div class="question-wrapper">
+            <div class="question">
+                Q2. Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur iste quod, quis excepturi ?
+            </div>
+            <div class="answer">
+                <button class="btn outline">lorem</button>
```
Comment (tarunsankhla) [inline Pages/BigBangQuiz/BigBangAnswers.html:40]: Just a suggestion you could have used list and in that button it get easier to view hem

File: Pages/Category/category.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+<body>
+    <div class="wrapper">
+        <div>
+            <h1 class="heading">Quiz Category</h1>
+
+        </div>
+        <div class="flex-row">
+            <div>
+                <div class="card card-box-shadow">
```
Comment (tarunsankhla) [inline Pages/Category/category.html:21]: Add comments in code

File: index.html [hunk 3]
```diff
@@ -6,10 +6,34 @@
...
+            <h1 class="heading">Quiz</h1>
+
+        </div>
+        <div>
+            <h2>Rules</h2>
+        </div>
+        <div>
+            <ul class="rules">
+                <li>First you select a category from the given categories.</li>
+                <br>
```
Comment (tarunsankhla) [inline index.html:25]: Avoid using br

**Review by tarunsankhla — APPROVED** [review]
Overall classname and UI is great, just minor changes

File: Pages/BigBangQuiz/BigBang.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                Q1. Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur iste quod, quis excepturi ?
+            </div>
+            <div class="answer">
+                <button class="btn outline">lorem</button>
+                <button class="btn outline">lorem</button>
+                <button class="btn outline">lorem</button>
+                <button class="btn outline">lorem</button>
+            </div>
+
+
```
Comment (HirenChafekar) [inline Pages/BigBangQuiz/BigBang.html:32]: remove extra spaces

File: Pages/BigBangQuiz/BigBang.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        </div>
```
Comment (HirenChafekar) [inline Pages/BigBangQuiz/BigBang.html:47]: remove extra spaces

**Review by m26unkwn — APPROVED** [review]
Things which i think good.
1. code quality.(you could have used semantic html)
2. Naming conventions (easy to understand)

Things which can be improved

1. Try to find which thing is repeating most like e.g margin :1rem
- put it in variable
2. Create classes for repeating properties i mean like flex.

Otherwise great work.

**Review by HirenChafekar — APPROVED** [review]

File: Pages/BigBangQuiz/BigBang.css [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+.answer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  margin-top: 2rem;
+}
+
+.btn {
```
Comment (govindappaarun) [inline Pages/BigBangQuiz/BigBang.css:21]: As its a  common component, styles will apply to all buttons, which may not be desired, so scope to a parent

File: Pages/Category/category.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.heading {
+  font-size: 2rem;
+  color: var(--primary-color);
+}
+.card-btn {
```
Comment (govindappaarun) [inline Pages/Category/category.css:12]: add consistent spacing between declarations

File: Pages/Category/category.css [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  text-align: center;
+}
+
+.add-to-cart {
+  width: 80%;
+}
+
+.card-image {
+  width: 32rem;
+  /* width: auto; */
```
Comment (govindappaarun) [inline Pages/Category/category.css:22]: remove comments

File: css/main.css [hunk 9]
```diff
@@ -0,0 +1,26 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```
Comment (govindappaarun) [inline css/main.css:5]: these lines are repetitive, you can move to utilities or to a common style file

File: index.html [hunk 10]
```diff
@@ -6,10 +6,34 @@
...
+                <li>
+                    The quiz for selected category would have 5 questions and each question holds 10 points.</li>
+                <br>
+
+                <li>The check your result.</li>
+            </ul>
+        </div>
+
+        <button onclick="javascript:window.location.href='../Pages/Category/category.html'" class="btn">Take the Quiz
+            ></button>
```
Comment (govindappaarun) [inline index.html:35]: syntax wrong, are you trying to add gt symbol here?

File: Pages/FriendQuiz/friendsquiz.html [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+                <button class="btn outline">lorem</button>
+                <button class="btn outline">lorem</button>
+            </div>
+
+
+        </div>
+
+        <div class="card-btn">
+            <button onclick="javascript:window.location.href='./friendanswers.html'" class="add-to-cart">Check
+                your score ></button>
```
Comment (govindappaarun) [inline Pages/FriendQuiz/friendsquiz.html:90]: use icon or html entities

**Review by govindappaarun — COMMENTED** [review]
Please fix some of those important ones from the comments list

File: Pages/BigBangQuiz/BigBangAnswers.html (hunk 1 above)
Comment (AagmanSuri) [reply Pages/BigBangQuiz/BigBangAnswers.html:40]: Thanks for the review [tarunsankhla](https://github.com/tarunsankhla)

**Pull request merged**